zoomus/components/__init__.py

```diff
@@ -2,4 +2,5 @@
 
 from __future__ import absolute_import
 
-from . import meeting, past_meeting, recording, report, user, webinar
+
+from . import meeting, past_meeting, recording, report, user, webinar, live_stream
```

Comment:
If you take a look at what is in develop now, these are now in alphabetical order. When you merge develop back in to this branch, can you please put `live_stream` at the appropriate place? Thanks.

Reply:
This is also updated